services/gateway/apis/http-version3/methods/modules/nft/nftEscrowed.js

```diff
@@ -0,0 +1,54 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2023 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const nftEscrowedSource = require('../../../../../sources/version3/nftEscrowed');
+const envelope = require('../../../../../sources/version3/mappings/stdEnvelope');
+const regex = require('../../../../../shared/regex');
+const { transformParams, response, getSwaggerDescription } = require('../../../../../shared/utils');
+
+module.exports = {
+	version: '2.0',
+	swaggerApiPath: '/nft/escrowed',
+	rpcMethod: 'get.nft.escrowed',
+	tags: ['NFT'],
+	params: {
+		escrowedChainID: { optional: true, type: 'string', pattern: regex.CHAIN_ID, altSwaggerKey: 'nftEscrowChainID' },
```

Comment:
Let's rename this property to `escrowChainID` instead.
```suggestion
		escrowChainID: { optional: true, type: 'string', pattern: regex.CHAIN_ID, altSwaggerKey: 'nftEscrowChainID' },
```